sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2174,6 +2174,15 @@ def init(self):
 
         self.log_notice("XCVRD INIT: After port config is done")
         port_mapping_data = port_event_helper.get_port_mapping(self.namespaces)
+        # remove ports from TRANSCEIVER_INFO table, if they don't exist in CONFIG DB
```

Comment:
@noaOrMlnx Can we move this block to a separate function?

Reply:
@mihirpat1 Done